modules/vm/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "name" {
```

Comment:
add some outputs of VM module to provide name, id, and private IP address of each created VM